src/Hazelcast.Net/Networking/AddressProvider.cs

```diff
@@ -169,7 +210,19 @@ internal static IEnumerable<IPAddress> GetV6LocalAddresses()
             }
         }
 
-        internal IEnumerable<NetworkAddress> ExpandPorts(NetworkAddress address, IPAddress ipAddress = null)
+        /// <summary>
+        /// Expands the port of an address.
+        /// </summary>
+        /// <param name="address">The address.</param>
+        /// <param name="ipAddress"></param>
+        /// <returns>Addresses with ports.</returns>
+        /// <remarks>
+        /// <para>If the <paramref name="address"/> has a specified port, this yields the address, and
+        /// only the address. But if it does not have a specified port, this yields the address with
+        /// different ports obtained through the <see cref="NetworkingOptions.DefaultPort"/> and
+        /// <see cref="NetworkingOptions.PortRange"/> configuration options.</para>
+        /// </remarks>
+        private IEnumerable<NetworkAddress> ExpandPorts(NetworkAddress address, IPAddress ipAddress = null)
```

Comment:
I think that would be great to have log here whether ports are expended or not. We struggled about that production side of a client last week. -> #543 

But we can save it to issue #543. I just wanted to mention when I saw this method.

Reply:
Let's do it in #543 yes. Wasn't aware of the issue.